heat-stack/app/utils/constants.ts

```diff
@@ -1,3 +1,15 @@
 // DO NOT USE ENV VARIABLES FOR FROM EMAIL -
 // INTENTIONALLY HARDCODED FOR SECURITY
+
 export const EMAIL_FROM = 'no-reply@heat.heatsmartalliance.org'
+
+// CACHE_DATABASE_PATH: from env, or default if not PROD
+export const CACHE_DATABASE_PATH = (() => {
```

Comment:
Ensemble says: Please, See if there is some other way to provide a .env file that is the test one, without putting this in the main code as a permanent workaround.